CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Features
 
+- Remove deprecated function `pybamm_install_jax` and renamed `have_jax` to
```

Comment:
Should be in "Breaking changes" or a "Misc" category? Definitely not in "Features"

Reply:
Ok I will switch it to a different section

Reply:
Done